ocrd_eval.schema.yml

```diff
@@ -0,0 +1,192 @@
+
+      There are two OCR-D *workflows* involved:
+        - ocr_workflow: The workflow which produced the OCR results to evaluate
+        - eval_workflow: The workflow run to evaluate OCR and GT
+
+      There are three OCR-D *workspaces* involved:
+        - gt_workspace: The workspace containing the GT
+        - ocr_workspace: The workspace containing the OCR results from ocr_workflow
+        - eval_workspace: The workspace on which the eval_workflow was run
+
+    required:
+      - ocr_workflow
+      - ocr_workspace
+      - eval_workflow
+      - eval_workspace
+      - gt_workspace
+      - document_metadata
+
+    properties:
+
+      ocr_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the ocr_workspace
+
+      ocr_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the OCR
+
+      eval_workflow:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The OCR-D workflow that produced the eval_workspace
+
+      eval_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the evaluation results
+
+      gt_workspace:
+        allOf:  [{ '$ref': '#/$defs/LabeledUrl' }]
+        description: The workspace containing the GT
+
+      workflow_steps:
+        type: object
+        description: Human readable description of the individual steps in the workflow (for UI)
+        patternProperties:
+          '^[0-9]+$':
+            type: string
+            description: Description of this workflow step
+
+      workflow_model:
+        type: string
+        description: Human readable name of the main model used for recognition in the OCR workflow (for UI)
+
+      eval_tool:
+        type: string
+        description: Human readable name and version of evaluation tool used (for UI)
+
+      document_metadata:
+        type: object
+        title: Bibliographical and typographical metadata about the work to be evaluated
+        properties:
+
+          publication_year:
+            type: number
+            description: Year the document was originally published
+
+          publication_century:
+            type: string
+            description: Century the document was originally published
+            pattern: '[12][0-9]{3}-[12][0-9]{3}'
+
+          publication_decade:
+            type: string
+            description: Decade the document was originally published
+            pattern: '[12][0-9]{2}0-[12][0-9]{2}0'
+
+          number_of_pages:
+            type: number
+            description: Number of pages in this work (i.e. the number of images in the gt_workspace)
+
+          layout:
+            type: string
+            enum: ['simple', 'complex']
+
+          fonts:
+            type: array
+            items:
+              type: string
+              enum: ['antiqua', 'fraktur']
```

Comment:
```suggestion
              enum: ['Antiqua', 'Black Letter']
```

Reply:
These are the values I conveniently get from the data, but we could stick to the draft, too. I just need to know.
Is `fraktur` used more widely than `Black Letter`, btw?

Reply:
`Black letter` is equivalent to "gebrochene Schriften", of which "Fraktur" is but one - used in prints rather than manuscripts like others such as "Textura" or "Bastarda", cf. https://github.com/OCR-D/ocrd_typegroups_classifier#colwise-classifier

So "Black Letter" is more general but in this case, we do actually mean "Fraktur" AFAIK the GT data (@tboenig - do we have non-Fraktur black letter in our data?).

Reply:
Thanks for elaborating!

In the METADATA.yml we have

```yaml
description: >-
  Ground Truth with a collection of documents with the following
  characteristics: font blackletter
```
and 

```yaml
script:
  - Goth
```
I re-checked the way the font classification is done and realized that I could just as easily use `Fraktur`. So let's go for that if @tboenig doesn't have any objections.

Reply:
@tboenig is for fine-grained font description with the list of values from https://tboenig.github.io/gt-guidelines/html/trans/lySchriftarten.html

Reply:
No problem, I can add that.

Reply:
See [00a9020](https://github.com/OCR-D/spec/pull/236/commits/00a9020b02dafeb32b77f7512644bfe21e1f881b). Resolved?